MultiPartyExample/MainViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  MainViewController.swift
+//  MultiPartyExample
+//
+//  Copyright © 2019 Twilio, Inc. All rights reserved.
+//
+
+import UIKit
+import TwilioVideo
+
+class MainViewController: UIViewController {
+
+    // MARK: View Controller Members
+
+    // Configure access token manually for testing, if desired! Create one manually in the console
+    // at https://www.twilio.com/console/video/runtime/testing-tools
+    var accessToken = "TWILIO_ACCESS_TOKEN"
+
+    // Configure remote URL to fetch token from
+    var tokenUrl = "http://localhost:8000/token.php"
+
+    // Maximum bitrate (in kbps) used to send video.
+    static let kMaxVideoBitrate = UInt(1500)
+
+    // MARK: UI Element Outlets and handles
+    @IBOutlet weak var connectButton: UIButton?
+    @IBOutlet weak var roomTextField: UITextField?
+    @IBOutlet weak var roomLine: UIView?
+    @IBOutlet weak var roomLabel: UILabel?
+    @IBOutlet weak var messageLabel: UILabel?
+
+    // MARK: UIViewController
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        messageLabel?.adjustsFontSizeToFitWidth = true
+        messageLabel?.minimumScaleFactor = 0.75
+        connectButton?.layer.cornerRadius = 4
+
+        let tap = UITapGestureRecognizer(target: self, action: #selector(MainViewController.dismissKeyboard))
+        view.addGestureRecognizer(tap)
+
+        /*
+         * Choose default settings that are appropriate for a multi-party Group Room.
+         * In order to ensure good quality of service for all users, the Client prefers VP8 simulcast.
+         * Since the video being shared is VGA the Client restricts the amount of bandwidth used for publishing video.
+         */
+        Settings.shared.videoCodec = TVIVp8Codec(simulcast: true)
+        Settings.shared.maxVideoBitrate = 1024 * MainViewController.kMaxVideoBitrate
+
+        roomTextField?.becomeFirstResponder()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        roomTextField?.text = ""
+        logMessage(messageText: "Twilio Video v\(TwilioVideo.version())")
+    }
+
+    @objc func dismissKeyboard() {
+        if let textField = self.roomTextField,
+            textField.isFirstResponder {
+            textField.resignFirstResponder()
+        }
+    }
+
+    func logMessage(messageText: String) {
+        NSLog(messageText)
+        messageLabel?.text = messageText
+    }
+
+    @IBAction func connect(_ sender: Any) {
+        // An empty name is allowed, in order to support tokens which are scoped to a single Room.
+        guard let roomName = roomTextField?.text else {
```

Comment:
If we are supporting empty room names, does it make sense to even have this check here anymore?